Building.md

```diff
@@ -0,0 +1,49 @@
+This is the building guide for FullScreenShenanigans projects.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play a FullScreenShenanigans game, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the project repository using GitHub:
+1. If you don't already have Git installed on your computer, go to [this page](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instructions to download Git.
+2. Once Git is downloaded, follow [these directions](https://help.github.com/articles/cloning-a-repository/) to clone the repository.
```

Comment:
Should mention the URL format for FSS projects (.../FullScreenShenanigans/<project-name>/...)